pkg/scheduler/scheduler.go

```diff
@@ -69,7 +69,8 @@ type Scheduler struct {
 	// by NodeLister and Algorithm.
 	SchedulerCache internalcache.Cache
 
-	Algorithm ScheduleAlgorithm
+	// Schedule returns the result of Schedule
```

Comment:
This comment looks weird. Should be `return the result of scheduling`?

Reply:
Thanks for your opinion!